### ISSUE TITLE
[Bug] Field's `labelTooltip` not announced by screen readers

### ISSUE THREAD

**Issue by GuillaumeMMM** [facebook/astryx#6413]
### Description

The `Field`'s `labelTooltip` is not announced by screen readers. It is not announced when reaching the field by focusing into it, and the icon cannot be reached by navigating with the screen reader.

Since it is used as an additional description for the field, I would expect the tooltip content to be automatically announced right after the accessible name of the field when focusing into it.

### Reproduction

1. Render `<TextInput label='My label' value='' labelTooltip='My tooltip' />`
2. Tab to the field with the screen reader on
3. The screen reader only announces "My label"

### Astryx Version

@astryxdesign/core@0.6.2

### Environment

- MacOS 26.5.2 + Voiceover
- Google Chrome 153.0.8010.53
